react-notes/src/components/Notepad/Notepad.jsx

```diff
@@ -1,22 +1,40 @@
 import PropTypes from 'prop-types';
 
-import { NO_SELECTED_NOTES } from '../../config/constants';
-import { WrapperStyled, NotepadStyled, NotepadTop, NotepadPapper } from './styled';
+import { WrapperStyled, NotepadStyled, NotepadTop, NotepadPapper, NotepadTitle } from './styled';
+import { NO_SELECTED_NOTES } from '../../config/constants/index';
+import ToolbarContainer from '../Toolbar/toolbarContainer';
 
-const Notepad = ({ note: { title, content, date } }) => {
+const Notepad = ({ note, isEdit, setEdit, setSave, setText, setTitle, onDelete }) => {
   return (
     <WrapperStyled className="wrapper">
-      <NotepadStyled className="notepad">
-        <NotepadTop className="notepad__title">
-          <h3>{title || null}</h3>
-          <span>{date || null}</span>
+      <NotepadStyled className="notepad" onClick={(e) => e.stopPropagation()}>
+        <NotepadTop className="notepad__top">
+          <NotepadTitle
+            className="notepad_title"
+            contentEditable={isEdit}
+            suppressContentEditableWarning
+            onInput={(e) => setTitle(e)}
+          >
+            {note ? note.title : null}
+          </NotepadTitle>
+          <span>{note ? note.date : null}</span>
         </NotepadTop>
         <NotepadPapper
           className="notepad__papper"
-          contentEditable="true"
-          data-placeholder={content || NO_SELECTED_NOTES}
-        />
+          contentEditable={isEdit}
+          suppressContentEditableWarning
+          onInput={(e) => setText(e)}

```

Comment:
All the params will send as well in arrow function:
```suggestion
          onInput={setText}

```